### PR TITLE
Default NFS storageclass

### PR DESCRIPTION
**Pull request by mdbooth**
Image registry playbook is updated to use it.

### CODE REVIEW

File: ansible/roles/export_client_to_controller/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- name: Include variables
+  include_vars: vars/default.yaml
+
+# We use mkdir instead of file below because the local shell always expands ~
+# correctly locally, and doesn't try to change the permissions on the local
+# directory to the remote user
+
+- name: Ensure local directories exist
+  shell: |
+    mkdir -p {{ local_working_dir }}
```
Comment (slagle) [inline ansible/roles/export_client_to_controller/tasks/main.yaml:15]: where is local_working_dir defined?

File: ansible/vars/default.yaml [hunk 2]
```diff
@@ -68,3 +68,9 @@ mariadb_storage_worker: worker-0
 
 # worker that will host the storage for rabbitmq
 rabbitmq_storage_worker: worker-0
+
+local_working_dir: "~/{{ ocp_cluster_name }}-working"
```
Comment (slagle) [inline ansible/vars/default.yaml:72]: ah, i see it here :)

Comment (slagle) [inline ansible/vars/default.yaml:72]: i see it here

File: ansible/roles/export_client_to_controller/tasks/main.yaml (hunk 1 above)
Comment (mdbooth) [reply ansible/roles/export_client_to_controller/tasks/main.yaml:15]: Incidentally, I can't remember if I left a comment in, but the mkdir in shell works around ansible weirdness with connection: local. I'd like to think there's a better way to do this in ansible, but this the best I came up with.

File: ansible/vars/default.yaml (hunk 2 above)
Comment (mdbooth) [reply ansible/vars/default.yaml:72]: Yeah, I intend it to be wider scoped than the role. The nfs task puts yamls in there too.

File: ansible/files/ocp/namespace/namespace.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: openstack
```
Comment (dprince) [inline ansible/files/ocp/namespace/namespace.yaml:4]: Installing our operators via OLM will create the namespace for us (via the OperatorGroup). I'm not sure we'll need this namespace.yaml file.

File: ansible/olm.yaml [hunk 4]
```diff
@@ -21,60 +25,35 @@
...
 
   - name: build the csv-merger
     shell: >
       make csv-merger
     args:
-      chdir: "{{ base_path }}/github.com/openstack-k8s-operators/openstack-cluster-operator"
-      creates: "{{ base_path }}/github.com/openstack-k8s-operators/openstack-cluster-operator/build/_output/csv-merger"
+      chdir: "{{ repo_dir }}"
 
+  # XXX: This may not rebuild correctly as 'creates' only tests existence.
```
Comment (dprince) [inline ansible/olm.yaml:50]: The comment here doesn't quite tell the whole story as even if you used a Makefile the csv_version would still be the same. The intent here is to encourage you to bump csv_version if you are building a new OLM catalog and bundle. So lets do just that... "TODO: fail if this directory exists with an error message informing the user to bump the csv_version"

**Pull request merged**